package.json

```diff
@@ -72,7 +72,8 @@
     "test:unit": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=600 --no-check-coverage \"packages/cactus-*/src/test/typescript/unit/\"",
     "test:benchmark": "tap --ts --jobs=1 --no-timeout --no-check-coverage \"packages/cactus-*/src/test/typescript/benchmark/\"",
     "test:browser": "karma start karma.conf.js",
-    "test:integration": "tap --ts --node-arg=--max-old-space-size=4096 --jobs=1 --timeout=3600 --no-check-coverage \"packages/cactus-*/src/test/typescript/integration/\"",
+    "test:integration": "tap --ts --node-arg=--max-old-space-size=4096 --jobs=1 --timeout=3600 --no-check-coverage \"packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/\"",
+    "test:jestintegration": "jest --node-arg=--max-old-space-size=4096 --jobs=1 --timeout=3600 --no-check-coverage \"packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/\"",
     "changelog": "conventional-changelog --infile CHANGELOG.md --outfile CHANGELOG.md && git add CHANGELOG.md",
```

Comment:
@micoferdinand98 This looks like it was just left here by mistake, if so, please remove from the diff! LGTM otherwise. Thank you!

Reply:
@micoferdinand98 Are you still working on this?